job-templates/kubernetes_release_staging.json

```diff
@@ -33,7 +33,9 @@
     ],
     "windowsProfile": {
       "adminUsername": "azureuser",
-      "adminPassword": "replacepassword1234$"
+      "adminPassword": "replacepassword1234$",
+      "windowsDockerVersion": "18.09.2",
+      "sshEnabled": true
```

Comment:
I'm wondering how this works - this is something Azure-specific, right?

I actually have a PR pending for enabling OpenSSH for Windows nodes in our GCE test clusters: https://github.com/kubernetes/kubernetes/pull/74360. It includes a pretty lengthy script to download, install and configure Win32-OpenSSH; if you know of any easier ways to achieve this please feel free to comment on that PR :)